<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dis_openfile</class>
 <widget class="QWidget" name="dis_openfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打开文件</string>
  </property>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_con">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pre">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>521</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
